### DISCUSSION TITLE
iPad 9th gen - SEB menu not available, iPads locked

### DISCUSSION THREAD

**Discussion by ppttry** [SafeExamBrowser/seb-mac#339] [General]
Hi
Anyone had this problem before? We have used SEB quite extensively in our exam procedures, but now we encountered a new problem. The SEB does not show the menu when swiping from left edge of the ipad and therefore all ipads are now locked in exam environment and cannot be logged out from seb. What could be done to release the ipads?
All ipads have 16+ iPadOS version and lates SEB installed.

**Comment by ppttry** [comment]
We found one solution that does not make sense. We needed to logout from the exam environment (basically the website where we have locked the ipads), then the swipe from left menu worked, but that does not make any sense why it is like that the left menu is not available when logged in the exam portal?

**Comment by danschlet** [comment]
Swiping from left edge to reveal the side menu only works when the displayed web page isn't zoomed. You could try to zoom back to the default zoom level with two fingers pinch-to-zoom, then the left swipe should work. If not, then probably the website uses some non-standard CSS or zooms automatically.

If you're using external keyboards, the key shortcut cmd-M also shows the side menu.

The easiest is to just use default settings for the SEB Dock to be displayed on the bottom of the screen (see Edit Settings / User Interface / SEB Dock / Show Dock or settings key showTaskBar = true). Then tapping the SEB icon on the left shows the side menu in any case. Also you can use the "Shutdown" quit icon to exit a SEB (exam) session. The quit button can also be hidden in the before mentioned SEB Dock settings page, in case you prefer to have it only in the side menu.

**Comment by ppttry** [comment]
Thanks, we will keep this in mind, zoom might be the issue -> the exam environment resets the zoom when logging out and thats why it worked. will investigate, thanks for the reply!
